recipe/meta.yaml

```diff
@@ -15,13 +15,13 @@ package:
 
 source:
   url: https://github.com/esmf-org/esmf/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: dc270dcba1c0b317f5c9c6a32ab334cb79468dda283d1e395d98ed2a22866364
+  sha256: d7ab266e2af8c8b230721d4df59e61aa03c612a95cc39c07a2d5695746f21f56
   patches:
     # remove these very long running tests only for aarch64 nompi (which is hanging)
     - remove-long-running-tests.patch  # [aarch64 and mpi == 'nompi']
     # Sequence of Windows targeting patches; should be unix compatible
     - 0001-Add-x86_64-machine-id-for-MinGW.gfortran.patch
-    - 0002-Change-long-int-to-long-long-for-Windows-compatibili.patch
+    # - 0002-Change-long-int-to-long-long-for-Windows-compatibili.patch
```

Comment:
This one seemed to cause trouble on OSX (64 and ARM).  It may be that I messed up the update on the patch but since we're not building for Windows right now I'm just going to comment it out for now.

Reply:
Nope, I spoke too soon.  Same issue without the patch.